pkg/controller/node/node_controller.go

```diff
@@ -386,6 +395,17 @@ func (ctrl *Controller) addNode(obj interface{}) {
 		return
 	}
 
+	mastersSchedulable, err := ctrl.getMastersSchedulable()
+	if err != nil {
+		goerrs.Wrap(err, "Getting scheduler config failed")
+		return
+	}
+	err = ctrl.reconcileSchedulableMasters(mastersSchedulable)
```

Comment:
Why to make all masters schedulable when add a node? we can check if the current node is master and then make it schedulable.

Reply:
Makes sense - I guess I was taking the lazy brute force approach and re-using the existing code.  I'll update this